site/content/kapp-controller/_index.html

```diff
@@ -3,49 +3,65 @@
+      </div>
     </div>
+  </div>
 </div>
 
 <div class="wrapper mobile-blue">
-    <div class="yaml-grid grid three">
-        <div class="col">
-            <div class="icon">
-                <img src="/img/structure-oriented.svg" />
-            </div>
-            <h3>Consistent</h3>
-            <p>App CR provides a common consistent API for configuring and deploying your applications regardless if it is a Helm chart, plain YAML configuration, ytt templates, etc.</p>
-        </div>
-        <div class="col">
-            <div class="icon">
-                <img src="/img/modularized-config.svg" />
-            </div>
-            <h3>Composable</h3>
-            <p>Take advantage of different tools for each stage -- fetch, template and deploy. For example, fetch Helm templates from a Git repository, use Helm to template them, make modifications with ytt overlays, and finally deploy with kapp.</p>
-        </div>
-        <div class="col">
-            <div class="icon">
-                <img src="/img/deterministic.svg" />
-            </div>
-            <h3>Lightweight</h3>
-            <p>kapp-controller delegates all the heavy lifting to existing configuration management and deployment tools. In doing so, kapp-controller itself is kept very simple, and therefore provides a high degree of reliability.</p>
-        </div>
+  <div class="yaml-grid grid three">
+    <div class="col">
+      <div class="icon">
+        <img src="/img/structure-oriented.svg" />
+      </div>
+      <h3>App Developers</h3>
+      <p>Build your application for Kubernetes and let kapp-controller continuously fetch configuration and images 
+        from a variety of sources (git, helm charts, OCI image registry, etc.), template using various tools 
+        (ytt, kbld, helm, etc.), and deploy out to a cluster. 
+      </p>
     </div>
+    <div class="col">
+      <div class="icon">
+        <img src="/img/modularized-config.svg" />
+      </div>
+      <h3>Package Authors</h3>
+      <p> Encapsulate, version, and distribute your software for others to install on a Kubernetes cluster. 
+        Share your software packages with users in the same cluster where they are developing. 
+      </p>
+    </div>
+    <div class="col">
+      <div class="icon">
+        <img src="/img/deterministic.svg" />
+      </div>
+      <h3>Package Consumers</h3>
+      <p>Discover, configure, and install versioned software on a Kubernetes cluster without needing 
+        to know all the underlying details. kapp-controller will continuously fetch, template, and deploy 
+        the installed package just like an application.</p>
+    </div>
+  </div>
 </div>
 
 <div class="wrapper">
-    <div class="text-content" id="examples">
-        <h2>Basic Usage</h2>
-        <p>Describe in <code>simple-app.yml</code> how to fetch, template, and deploy your application:</p>
-        {{< highlight yaml "" >}}
+  <div class="text-content" id="examples">
+    <h2>Basic Usages</h2>
```

Comment:
i think we should have a bulleted list that indicates upcoming sections:

- App CR...
- Package authoring
- Package consumption